3rd-party/class.jetpack-amp-support.php

```diff
@@ -78,6 +78,11 @@ function_exists( 'amp_is_canonical' ) // this is really just testing if the plug
 		return apply_filters( 'jetpack_is_amp_request', $is_amp_request );
 	}
 
+	static function has_amp_suffix() {
+		$request_path = parse_url( $_SERVER['REQUEST_URI'], PHP_URL_PATH );
```

Comment:
You probably want to use `wp_parse_url()` instead of `parse_url()`. Prior to PHP 5.4, `parse_url()` didn't support relative URLs such as parsing the bare path. Alternatively, it would probably be better to use `strtok` instead:

```php
$request_path = strtok( $_SERVER['REQUEST_URI'], '?#' );
```

Reply:
👍